tests/ert/unit_tests/config/test_gen_kw_config.py

```diff
@@ -263,32 +262,32 @@ def test_gen_kw_is_log_or_not(
 )
 def test_gen_kw_distribution_errors(tmpdir, distribution, mean, std, error):
     with tmpdir.as_cwd():
-        config = dedent(
-            """
-        JOBNAME my_name%d
-        NUM_REALIZATIONS 1
-        GEN_KW KW_NAME template.txt kw.txt prior.txt
-        """
-        )
-        with open("config.ert", "w", encoding="utf-8") as fh:
-            fh.writelines(config)
+        template = "MY_KEYWORD <MY_KEYWORD>"
```

Comment:
Reason why this wasn't inlined too?

Reply:
That is because the template file name goes into storage and we don't want to mix in storage migration at this point.

Reply:
Ah, I understand now. Fixed it.